app/models/proposal.rb

```diff
@@ -148,6 +148,14 @@ def existing_observation_for(user)
     observations.find_by(user: user)
   end
 
+  def eligible_observers
+    if observations.count > 0
+      User.where(client_slug: client).where('id not in (?)', observations.pluck('user_id'))
```

Comment:
no, we can't. If there are zero observations, the generated SQL looks like:
```sql
select * from users where client_slug="foo" and id not in (null)
```
which will fail.